scripts/changePartnerShares.ts

```diff
@@ -0,0 +1,47 @@
+import { BigNumberish } from 'ethers';
+import { HardhatRuntimeEnvironment } from 'hardhat/types';
+import { ICollector } from 'typechain-types/contracts/Collector';
+import { parseJsonFile } from './utils';
+
+export type ChangePartnerSharesArg = {
+  collectorAddress: string;
+  partnerConfig: string;
+  gasLimit?: BigNumberish;
+};
+
+export type PartnerConfig = {
+  collectorOwner: string;
+  partners: ICollector.RevenuePartnerStruct[];
+  tokenAddress: string;
+};
+
+const defaultTxGas = 150000;
```

Comment:
I suggest using the convention for calling this kind of constants. Something like DEFAULT_TX_GAS.